message.go

```
@@ -1236,3 +1262,51 @@ func (m *RedisMessage) approximateSize() (s int) {
 	}
 	return
 }
+
+// String returns human-readable representation of RedisMessage
+func (m *RedisMessage) String() string {
+	v, err := (*prettyRedisMessage)(m).MarshalJSON()
+	if err != nil {
```

We can just ignore the err here.

```suggestion
```